application.bs

```diff
@@ -0,0 +1,2350 @@
+          [[REMOTE-PLAYBACK|Remote Playback API]] in an interoperable fashion.
+
+Group: secondscreenwg
+Markup Shorthands: markdown yes, dfn yes, idl yes, markup yes
+</pre>
+
+<pre class="anchors">
+url: https://html.spec.whatwg.org/multipage/media.html#concept-mediaerror-code; type: dfn; spec: html
+    text: media error code
+urlPrefix: https://html.spec.whatwg.org/multipage/media.html#; type: dfn; spec: html
+    text: official playback position
+    text: poster frame
+    text: timeline offset
+    text: media resource
+    text: media timeline
+urlPrefix: https://www.w3.org/TR/presentation-api/#dfn-; type: dfn; spec: PRESENTATION-API
+    text: available presentation display
+    text: controlling user agent
+    text: presentation; url: receiving-browsing-context
+    text: presentation identifier
+    text: presentation request url
+    text: receiving user agent
+urlPrefix: https://w3c.github.io/remote-playback/#dfn-; type: dfn; spec: REMOTE-PLAYBACK
+    text: availability sources set
+    text: compatible remote playback device
+    text: initiate remote playback
+    text: media element state
+    text: remote playback devices
+    text: remote playback source
+url: https://datatracker.ietf.org/doc/html/rfc9000#name-variable-length-integer-enc; type: dfn; spec: RFC9000; text: Variable-Length Integer Encoding
+url: https://datatracker.ietf.org/doc/html/rfc9000#name-variable-length-integer-enc; type: dfn; spec: RFC9000; text: variable-length integer
+url: https://tools.ietf.org/html/rfc5646#section-2; type: dfn; spec: RFC5646; text: language tag
+url: https://tools.ietf.org/html/rfc4122#section-4.4; type: dfn; spec: RFC4122; text: UUID
+url: https://tools.ietf.org/html/rfc6381#section-3; type: dfn; spec: RFC6381; text: codecs parameter
+url: https://tools.ietf.org/html/rfc8610#section-3; type: dfn; spec: RFC8610; text: concise data definition language
+</pre>
+
+Introduction {#introduction}
+============================
+
+The Open Screen Application Protocol connects browsers to devices capable of
+rendering Web content for a shared audience.  Typically, these are devices like
+Internet-connected TVs, HDMI dongles, and smart speakers.
+
+The Open Screen Application Protocol is intended to be supported by a variety of
+connection technologies between browsers and devices.  There are specific
+requirements to allow browsers and devices to discover and authenticate to one
+another, but these requirements may be met by multiple possible implemenations.
+To maximize interoperability, browsers and devices should support the Open
+Screen Network Protocol, which provides a way for browsers and devices to
+discover, connect, and authenticate each other on a local area network.
+
+It allows a browser to present a URL, initiate remote playback of an HTML <l
+spec=html>[=media element=]</l>, and stream media data to another device.
+
+The Open Screen Protocol is intended to be extensible, so that additional
```

Comment:
Similarly, the "Open Screen Protocol" would no longer exist (or would need to be defined as the combination of the Open Screen Application Protocol and the Open Screen Network Protocol).
```suggestion
The Open Screen Application Protocol is intended to be extensible, so that additional
```

Reply:
Done